src/index.ts

```diff
@@ -73,19 +73,26 @@ async function validateEmail(
  * Get user access cards
  * @param config Connection settings
  * @param userId user ID from NetReady
+ * @param code authorization code from cookies
  * @returns { accessCard: boolean, proCard: boolean }, otherwise: <ErrorResponse>
  */
 async function accessCards(
   config: NetReadyConfig,
   userId: number,
+  code?: string,
 ): Promise<{
   accessCard: boolean,
   proCard: boolean,
   error: false
 } | ErrorResponse> {
   try {
+    const reqConfig = code
+      ? { headers: { Cookie: `${config.authCookie}=${code}'` } }
+      : undefined;
```

Comment:
Maybe not adding reqConfig to request might be better than adding undefined value?